### ISSUE TITLE
Multiple participants in a single c3d

### ISSUE THREAD

**Issue by jfslin** [pyomeca/biorbd#226]
I am interested in loading a c3d that has two participants with the same marker set to perform IK. Is this possible with biorbd as-is? Or should I export my c3d so that the two participants are in two different files and run IK for them separately? Thank you.

**Comment by pariterre** [comment]
Hi @jfslin 
`biorbd` won't manage by itself the makers from a C3D. In other words, you have to make sure beforehand that the vector of markers you send to biorbd is coherent with the marker list defined in the bioMod. 

If you are one of the binder (Python or Matlab) it is trivial as it only consists of slicing your data. If you are accessing via C++ it can be a bit more tricky as you have to format your stl_vector properly. 

In any case, that means your question is not necessarily relevant from the biorbd perspective, but it is relevant from the c3d reader's perspective you are using. Therefore, depending if you are using `ezc3d`, `pyomeca`, `BTK` or `OpenSim` to access your c3d file, the answer to your question will differ.

**Comment by jfslin** [comment]
Hi @pariterre, 

I am using `ezc3d` to load my c3d file, and using Python, and got it working. Documenting what I did here in case it is helpful:

- Starting with a general `.bioMod` file that has a complete model for a single participant
- Create a new `.bioMod` file that concatenated the two copies of the model of the above file. I prepended my marker names and segment names with the names of my two participants, to respect 'biorbd''s requirement that segment names are all unique
- Ensure that ensured that the marker order that I am passing into the EKF matched the order the model files

A quick check with `bioviz` seems pretty good. Thank you for responding.
